.travis.release.images.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+  
+set -xe
+
+OWNER="${OWNER:-radanalyticsio}"
+IMAGES="${IMAGES:-
+  openshift-spark
+  openshift-spark-py36
+  openshift-spark-inc
+  openshift-spark-py36-inc
+}"
+
+main() {
+  if [[ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]]; then
+    echo "Squashing and pushing the the :latest images to docker.io and quay.io"
+    installDockerSquash
+    loginDockerIo
+    pushLatestImages "docker.io"
+    loginQuayIo
+    pushLatestImages "quay.io"
+  elif [[ "${TRAVIS_TAG}" =~ ^[0-9]+\.[0-9]+\.[0-9]+-[0-9]+$ ]]; then
+    echo "Squashing and pushing the '${TRAVIS_TAG}' images to docker.io and quay.io"
+    installDockerSquash
+    loginDockerIo
+    pushReleaseImages "docker.io"
+    loginQuayIo
+    pushReleaseImages "quay.io"
+  else
+    echo "Not doing the docker push, because the tag '${TRAVIS_TAG}' is not of form x.y.z-n or we are not building the master branch"
+  fi
+}
+
+loginDockerIo() {
+  set +x
+  docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD"
+  set -x
+}
+
+loginQuayIo() {
+  set +x
+  docker login -u "$QUAY_USERNAME" -p "$QUAY_PASSWORD" quay.io
+  set -x
+}
+
+installDockerSquash() {
+  command -v docker-squash || pip install --user docker-squash
+}
+
+squashAndPush() {
+  if [[ $# != 3 ]]; then
+    echo "Usage: squashAndPush input_image output_image" && exit
+  fi
+  local _in=$1
+  local _out=$2
+
+  echo "Squashing $_out.."
+  # squash last 22 layers (everything up to the base centos image)
+  docker-squash -F 22 -t $_out $in
```

Comment:
i think this needs to be

`docker-squash -f 22 -t $_out $in`

my install of docker-squash is not showing `-F` as a valid flag.

from my testing:

```
$ docker-squash -F 22 -t openshift-spark:squashed openshift-spark:latest
usage: docker-squash [-h] [-v] [--version] [-d] [-f FROM_LAYER] [-t TAG] [-c]
                     [--tmp-dir TMP_DIR] [--output-path OUTPUT_PATH]
                     image

Docker layer squashing tool

positional arguments:
  image                 Image to be squashed

optional arguments:
  -h, --help            show this help message and exit
  -v, --verbose         Verbose output
  --version             Show version and exit
  -d, --development     Does not clean up after failure for easier debugging
  -f FROM_LAYER, --from-layer FROM_LAYER
                        ID of the layer or image ID or image name. If not
                        specified will squash all layers in the image
  -t TAG, --tag TAG     Specify the tag to be used for the new image. If not
                        specified no tag will be applied
  -c, --cleanup         Remove source image from Docker after squashing
  --tmp-dir TMP_DIR     Temporary directory to be created and used
  --output-path OUTPUT_PATH
                        Path where the image should be stored after squashing.
                        If not provided, image will be loaded into Docker
                        daemon

Error: unrecognized arguments: -F openshift-spark:latest
```

Reply:
good catch! it's a typo